modules/stanford_news_helper/stanford_news_helper.module

```diff
@@ -17,9 +17,9 @@ function stanford_news_helper_form_node_stanford_news_form_alter(&$form, FormSta
     $subform = &$form['field_s_news_media_contacts']['widget'][0]['subform'];
     $subform['field_highlight_cards_title']['widget'][0]['value']['#default_value'] = 'Media Contacts';
     $card_subform = &$subform['field_p_section_highlight_cards']['widget'][0]['subform'];
-    $card_subform['field_p_highlight_card_title']['widget'][0]['value']['#default_value'] = 'Josie Garthwaite';
-    $card_subform['field_p_highlight_card_subtitle']['widget'][0]['value']['#default_value'] = '650-497-0947';
-    $card_subform['field_p_highlight_card_desc']['widget'][0]['value']['#default_value'] = 'josieg@stanford.edu';
```

Comment:
How come the removal of the `['value']` key here?

Reply:
They were removed, but then new ones were added back in replacing Josie with Dee.

Reply:
If you keep it, it won't ever populate that desc field. I did this work a long time ago and forget the exact reference that led me to that solution, but seem to recall it has to do with it being a long text field.